src/patternfly/patternfly-base-breakpoint-minus-sidebar.scss

```diff
@@ -0,0 +1,8 @@
+$pf-global--exclude-sidebar-in-breakpoints: true;
```

Comment:
What's the use case for this file? This builds `patternfly-base.css` with the var set, but this stylesheet doesn't include media queries. It's just the global vars and such - https://unpkg.com/browse/@patternfly/patternfly@4.213.0/patternfly-base.css

Also if you wanted to build patternfly-base, you could do it like this by just importing the existing `patternfly-base.scss` like https://github.com/patternfly/patternfly/blob/67698913b57c147dfd6d517e6b79fe13e7979b70/src/patternfly/patternfly-base-no-reset.scss

Or if you wanted to create a `patternfly.css` with these changes, you'd just import `patternfly.scss` - https://github.com/patternfly/patternfly/blob/67698913b57c147dfd6d517e6b79fe13e7979b70/src/patternfly/patternfly-no-reset.scss

Reply:
Updated